07-routing-gossip.md

```diff
@@ -249,6 +249,10 @@ optional) features will have _odd_ feature bits, while incompatible features
 will have _even_ feature bits
 (["It's OK to be odd!"](00-introduction.md#glossary-and-terminology-guide)).
 
+A delay of 12-blocks is used when forgetting a channel on funding output spend
```

Comment:
I have a hard time parsing "channel on funding output spend". What is exactly meant by that?

Also, nit: s/12-blocks/12 blocks/

Reply:
See L226: https://github.com/lightning/bolts/pull/1004/files#diff-35876d484bc2647647dd95e16e18f0aa6c2b098b5f70834ec7fea33cdc522835R226